### DISCUSSION TITLE
Studying the Function of the SideDiffusiveFluxIntegral PostProcessor

### DISCUSSION THREAD

**Discussion by TLWise** [idaholab/moose#28968] [Q&A Modules: General]
For the SideDiffusiveFluxIntegral postprocessor a variable like vel_x is required. I have looked at the various associated .h and .C files and I do not see where the input for the variable is declared and applied. Which code file is the input and processor for the variable?

**Comment by GiudGiud** [comment]
Hello

The  SideIntegralVariablePostprocessor is the base class of `SideDiffusiveFluxIntegral ` that actually adds the variable parameter and processes it. It can then be used as an attribute in `SideDiffusiveFluxIntegral`.

The variable probably wont be 'vel_x' though, velocities are generally not "diffused"
